### ISSUE TITLE
Not working 

### ISSUE THREAD

**Issue by aruvee** [pdevty/googlefinance-client-python#6]
I think the Google Finance API is changed, so the version is not working. Any plans to update?

**Comment by andreylisovskiy** [comment]
Yes, same here. Nothing new from google, they always strive to make it horrible for developers, thats their motto. Thats they change API, deprecate it, or change the format. I get this error even when running the example code:
`....lib/python2.7/site-packages/googlefinance/client.py", line 15, in get_price_data
    basetime = int(cols[0][1:])
ValueError: invalid literal for int() with base 10: 'erial vehicles (UAV)'`

How can we mitigate that?

**Comment by thomashaselwanter** [comment]
Check the other thread for the fix: https://github.com/pdevty/googlefinance-client-python/issues/3

**Comment by andreylisovskiy** [comment]
@thomashaselwanter  Thank you.
